packages/components/tests/dummy/app/templates/components/text.hbs

```diff
@@ -0,0 +1,123 @@
+
+  <Shw::Text::H4 @tag="h3">Text::Display</Shw::Text::H4>
+  <Shw::Flex @direction="column" as |SF|>
+    {{#each this.model.DISPLAY_AVAILABLE_SIZES as |size|}}
+      <SF.Item @label="size={{size}}">
+        {{#let (get this.model.DISPLAY_AVAILABLE_WEIGHTS_PER_SIZE size) as |weights|}}
+          {{#each weights as |weight|}}
+            <Hds::Text::Display @size={{size}} @tag="p" @weight={{weight}}>The fox jumped over the lazy dog ({{weight}})</Hds::Text::Display>
+          {{/each}}
+        {{/let}}
+      </SF.Item>
+    {{/each}}
+  </Shw::Flex>
+
+  <Shw::Text::H4 @tag="h3">Text::Body</Shw::Text::H4>
+  <Shw::Flex @direction="column" as |SF|>
+    {{#each this.model.BODY_AVAILABLE_SIZES as |size|}}
+      <SF.Item @label="size={{size}}">
+        {{#let (get this.model.BODY_AVAILABLE_WEIGHTS_PER_SIZE size) as |weights|}}
+          {{#each weights as |weight|}}
+            <Hds::Text::Body @size={{size}} @tag="p" @weight={{weight}}>The fox jumped over the lazy dog ({{weight}})</Hds::Text::Body>
+          {{/each}}
+        {{/let}}
+      </SF.Item>
+    {{/each}}
+  </Shw::Flex>
+
+  <Shw::Text::H4 @tag="h3">Text::Code</Shw::Text::H4>
+  <Shw::Flex @direction="column" as |SF|>
+    {{#each this.model.CODE_AVAILABLE_SIZES as |size|}}
+      <SF.Item @label="size={{size}}">
+        {{#let (get this.model.CODE_AVAILABLE_WEIGHTS_PER_SIZE size) as |weights|}}
+          {{#each weights as |weight|}}
+            <Hds::Text::Code @size={{size}} @tag="p" @weight={{weight}}>The fox jumped over the lazy dog ({{weight}})</Hds::Text::Code>
+          {{/each}}
+        {{/let}}
+      </SF.Item>
+    {{/each}}
+  </Shw::Flex>
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H2>Alignment</Shw::Text::H2>
+  <Shw::Grid @columns={{3}} as |SG|>
+    {{#each this.model.AVAILABLE_ALIGNS as |alignment|}}
+      <SG.Item>
+        <Shw::Flex @direction="column" as |SF|>
+          <SF.Item @label="parent with text-align={{alignment}}">
+            <Shw::Outliner {{style text-align=alignment}}>
+              <Hds::Text::Body @size="200" @tag="p">The fox jumped over the lazy dog</Hds::Text::Body>
+            </Shw::Outliner>
+          </SF.Item>
+          <SF.Item @label="with @align={{alignment}}">
+            <Shw::Outliner>
+              <Hds::Text::Body @size="200" @tag="p" @align={{alignment}}>The fox jumped over the lazy dog</Hds::Text::Body>
+            </Shw::Outliner>
+          </SF.Item>
+        </Shw::Flex>
+      </SG.Item>
+    {{/each}}
+  </Shw::Grid>
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H2>Color</Shw::Text::H2>
+
+  <Shw::Text::H4 tag="h3">Color inheritance</Shw::Text::H4>
+  <Shw::Flex @direction="row" @wrap={{true}} {{style gap="2rem"}} as |SF|>
+    <SF.Item @label="parent with no specific color">
+      <div>
+        <Hds::Text::Body @size="300" @tag="p">Lorem ipsum dolor</Hds::Text::Body>
+      </div>
+    </SF.Item>
+    <SF.Item @label="parent with #e91e63 color">
```

Comment:
This is getting flagged by a11y automation.

> Element has insufficient color contrast of 4.34 (foreground color: #e91e63, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

It would be better if our pages themselves were conformant as possible, but looking past that, do we have documentation for our users on this potential issue?